docs/pages/build/building-from-github.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Trigger builds from the Expo GitHub App
```

Comment:
Do you think for the sidebar we shorten the title to "Trigger builds from GitHub app"? I don't have the context on "Expo GitHub App", so that's why I'm asking this question :)

Reply:
Yeah, shortening it makes sense